pom.xml

```diff
@@ -353,7 +353,8 @@
     <versions.google.cloud.core.http>2.23.0</versions.google.cloud.core.http>
 
     <versions.guava>32.0.1-jre</versions.guava>
-    <versions.google.gson>1.43.1</versions.google.gson>
+    <versions.google.http.client.gson>1.43.1</versions.google.http.client.gson>
+    <versions.google.gson>2.8.9</versions.google.gson>
```

Comment:
versions.google.gson was taken by google-http-client-gson, so I renamed the variable to a mathing name and using `versions.google.gson` for `com.google.gson`